src/app/technical-record/adr/adr-details/adr-details.component.html

```diff
@@ -14,7 +14,7 @@ <h2 class="govuk-heading-m">ADR details</h2>
       <dt class="govuk-summary-list__key">Date processed</dt>
       <dd id="test-ADR-approvalDate" class="govuk-summary-list__value">
         <span *ngIf="adrDetails.vehicleDetails?.approvalDate; else hyphen">
-          {{ adrDetails.vehicleDetails?.approvalDate | date: 'dd/MM/yyyy' }}
+          {{ adrDetails.vehicleDetails?.approvalDate | date: 'dd/MM/yyyy':'UTC' }}
```

Comment:
not every date field need UTC. Please check the API and only use on date fields with time stamp